ocp_resources/cluster_role.py

```diff
@@ -1,51 +1,31 @@
 # -*- coding: utf-8 -*-
-from ocp_resources.constants import TIMEOUT_4MINUTES
 from ocp_resources.resource import Resource
 
 
 class ClusterRole(Resource):
     """
     ClusterRole object
+    'rbac_authorization_k8s_io' API official docs:
+        https://docs.openshift.com/container-platform/3.11/rest_api/rbac_authorization_k8s_io/rbac-authorization-k8s-io-index.html
     """
 
     api_group = Resource.ApiGroup.RBAC_AUTHORIZATION_K8S_IO
 
-    def __init__(
-        self,
-        name=None,
-        client=None,
-        rules=None,
-        teardown=True,
-        yaml_file=None,
-        delete_timeout=TIMEOUT_4MINUTES,
-        **kwargs,
-    ):
+    def __init__(self, rules=None, **kwargs):
```

Comment:
Might have missed but why did we remove all init args? (e.g. yaml_file, name, client etc.)

Reply:
They'll be passed as kwargs and we're passing them forward in `super().__init__(**kwargs)` (line 26)